modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -589,29 +594,69 @@ export default class Tileset3D {
     this.stats.get(POINTS_COUNT).count = pointsRenderable;
   }
 
-  _initializeTileSet(tilesetJson) {
+  async _initializeTileSet(tilesetJson) {
+    if (this.type === TILESET_TYPE.I3S) {
+      // Calculate cartographicCenter & zoom props to help apps center view on tileset
+      this.calculateViewPropsI3S();
+      tilesetJson.root = await tilesetJson.root;
+    }
     this.root = this._initializeTileHeaders(tilesetJson, null);
 
-    // TODO CESIUM Specific
     if (this.type === TILESET_TYPE.TILES3D) {
       this._initializeCesiumTileset(tilesetJson);
+      // Calculate cartographicCenter & zoom props to help apps center view on tileset
+      this.calculateViewPropsCesium();
```

Comment:
Is this really Cesium specific, or is it just Tiles3D specific?

Reply:
It is Tiles3D specific. I just adhere existing naming